patchdeps.py

```diff
@@ -471,12 +471,12 @@ def analyze_hunk(self, depends, patch, hunk):
 
     def print_blame(self):
         print("{}:".format(self.fname))
-        next_line = None
+        next_line = 0
```

Comment:
I do not really like this change (commit " refactor: Make next_line an int "). I had intentionally used `None` to mean "no next line" instead of using 0, since it is more explicit and does not need to assume anything about the range of line numbers in line state (though this would actually require the check a few lines down to be `next_line is not None` instead of just `next_line`, I now realize).

Maybe drop this commit? AFAICS it is not needed for other changes?